src/sentry/auth/helper.py

```diff
@@ -738,7 +746,7 @@ def is_valid(self) -> bool:
 
     def get_initial_state(self) -> Mapping[str, Any]:
         state = dict(super().get_initial_state())
-        state.update({"flow": self.flow})
+        state.update({"flow": self.flow, "referrer": self.referrer})
```

Comment:
@nhsiehgit it'll be good to have some basic test coverage for the new `referrer` state attribute in https://github.com/getsentry/sentry/blob/master/tests/sentry/models/test_apitoken.py 😅

Reply:
🤔 
in test_apitoken?
Or did you mean in test_helper.py?